docs/docs/development/setup/setup.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Getting Started
+sidebar_label: Getting Started
+---
+
+## Environment Setup
+
+There are 2 ways to setup the zmk development environment: Docker+VS Code (Docker in the rest of the documentation) and using the operating system directly (native in the rest of the documentation). The Docker approach is a self-contained development environment while the native approach will setup your local operating system for development. The Docker approach is great for getting going quickly while the native approach is a bit faster but more difficult to setup initially.
```

Comment:
```suggestion
There are two ways to setup the zmk development environment: Docker+VS Code (Docker in the rest of the documentation) and using the operating system directly (native in the rest of the documentation). The Docker approach is a self-contained development environment while the native approach will setup your local operating system for development. The Docker approach is great for getting going quickly while the native approach is a bit faster but more difficult to setup initially.
```

It's usually better for readability to spell out small numbers like this. See https://www.grammarly.com/blog/when-to-spell-out-numbers/